### ISSUE TITLE
How do you compile the less files?

### ISSUE THREAD

**Issue by thepurpleblob** [bmbrands/theme_cerulean#5]
I notice that 'recess' no longer works because of deprecated features. 

Grunt is recommended for the parent 'bootstrap' theme but there is no Gruntfile.js for this child theme. Am I being stupid or is a suitable Gruntfile.js missing?

I'm assuming, of course, that one wants to modify the .less files.

**Comment by bmbrands** [comment]
That's right, you don't need recess or grunt to compile this theme's less files. Since Moodle 2.8 we can used the Moodle inbuilt php less compiler. 
On each change to the cerulean .less files you will need to purge all caches to see your changes. That's why there are no css files in the theme itself (other than the custom.css), they are re-generated when the theme caches are purged.

**Comment by thepurpleblob** [comment]
That's handy. I didn't know anything about it (as you may have guessed). Thanks!

**Comment by thepurpleblob** [comment]
For the benefit of anybody equally confused finding this ticket... I struggled to find any documentation about this feature (there's tons of conflicting documentation on themes), but eventually tracked it down here: https://docs.moodle.org/dev/Themes_overview#Compiling_LESS_on_the_fly

**Comment by tombola** [comment]
I note that the above documentation mentioned above states that "_Ideally your theme should precompile the LESS into CSS, but if you want to provide theme settings to your user, then using this feature is for you_"

I would personally prefer to use grunt to compile less, assuming it would be much faster than purging **all** caches. Will this cause me any issues?
